tasks/docker_exp.rb

```diff
@@ -7,17 +7,18 @@
 # TODO: detect what shell to use
 @shell_command = 'bash -lc'
 
-def provision(docker_platform, inventory_location)
+def provision(docker_platform, inventory_location, append_cli)
   include PuppetLitmus::InventoryManipulation
   inventory_full_path = File.join(inventory_location, 'inventory.yaml')
   inventory_hash = get_inventory_hash(inventory_full_path)
+  append_cli = " #{append_cli}"
 
   deb_family_systemd_volume = if (docker_platform =~ %r{debian|ubuntu}) && (docker_platform !~ %r{debian8|ubuntu14})
                                 '--volume /sys/fs/cgroup:/sys/fs/cgroup:ro'
                               else
                                 ''
                               end
-  creation_command = "docker run -d -it #{deb_family_systemd_volume} --privileged #{docker_platform}"
+  creation_command = "docker run -d -it #{deb_family_systemd_volume} --privileged #{docker_platform}#{append_cli}"
```

Comment:
How does this work even though there is no space between `#{docker_platform}` and `#{append_cli}`?

Reply:
line 14

Reply:
Ah I need to learn how to read :D